ratex-popup/popup.js

```diff
@@ -85,11 +85,16 @@ function popupInit(initialTime = 10000, followUpTime = 10000, duration = 5000) {
       "<div class=textContainer>" +
```

Comment:
you can replace all the quotation marks with template literals, which is easier to read and less prone to mistakes.